README.md

```diff
@@ -324,6 +324,96 @@ xsite:
       port: 7200
 ```
 
+Next the (local) port 7900 must be mapped to the port defined in the configuration (7200). On Docker for example by using `-p 7200:7900`
+
+Note: Caches won't be created automatically on the other side. So these must be created manually with the same name.
+
+So create a new cache either using the console (http://localhost:11222) or curl (or similar tool)
+`curl -XPOST -u <user>:<password> -H "Content-Type: application/xml" -d "@<path/to/cache.xml>" http://localhost:11222/rest/v2/caches/xsite-cache`
+  
+LON-Site:
+```
+<infinispan>
+  <cache-container>
+    <replicated-cache name="xsite-cache" statistics="true">
+      <backups>
+        <backup site="NYC" strategy="SYNC"/>
```

Comment:
The `backups` section should list all sites. "LON" should be included here.

Reply:
@pruivo This is correct, isn't it? `<backups>` must contain all sites that participate in the cross-site view.

Reply:
you are thinking about RELAY2. 
the cache configuration only contains the remote sites. If you put the local site, by mistake, it will be discarded.

Reply:
thanks, Pedro!